packages/core/image-asset/image-asset-common.ts

```
@@ -47,8 +47,30 @@ export function getAspectSafeDimensions(sourceWidth, sourceHeight, reqWidth, req
+			console.warn('Invalid width value provided: ', optionsCopy.width);
+			delete optionsCopy.width;
+		}
+	}
+
+	if (typeof optionsCopy.height === 'string') {
+		const parsedHeight = parseInt(optionsCopy.height, 10);
+		if (!isNaN(parsedHeight)) {
+			optionsCopy.height = parsedHeight;
+		} else {
+			console.warn('Invalid height value provided: ', optionsCopy.height);
+			delete optionsCopy.height;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)